Uchu.Core/Serializable/Struct/Property/PacketProperty.cs

```diff
@@ -31,7 +31,7 @@ public class PacketProperty : IPacketProperty
                         var data = (LegoDataDictionary) o;
                         if (data != null)
                         {
-                            var ldf = data.ToString();
+                            var ldf = data.ToString("\n");
```

Comment:
Did this change have any effects beyond leaderboards? Pretty sure there was another script somewhere that had 2 or more LDF entries.

Reply:
The player list you see when you load into AGS with 2+ players will now also highlight your own name correctly. I don't remember if there were any more places where this made a difference (but it didn't break anything in any case).